from neat.errors import *

class Error(Exception):
    pass

class RecordError(Error):
    pass

class TimeoutError(Error):
    pass
